addons/io_scene_gltf2/__init__.py

```diff
@@ -322,6 +322,8 @@ def __init__(self):
     scene_key = "glTF2ExportSettings"
 
     #
+    # Add a user extension list. It will be filled during the operator invocation
+    user_extensions = []
```

Comment:
The way this was before (on `self.user_extensions`) was correct. If you put it at the class level you're pushing new copies of stuff to it every time `invoke` gets called and you're also holding onto stuff that might have gotten unloaded because an addon was disabled etc.

Reply:
Would be valid if I add it as a class variable but I empty it in invoke before pushing the new extensions? I need it to be a class variable to pass CI for 279.

Reply:
Is the problem with storing it on `self`? Can you just collect the user extensions in `execute` instead of `invoke`?

Reply:
Fixed checking if the user_extensions attribute exist and assigning in that case